src/AspNetCoreCustom/Example/GraphQLMiddleware.cs

```diff
@@ -78,10 +70,10 @@ private async Task ExecuteAsync(HttpContext context)
 
         private async Task WriteResponseAsync(HttpContext context, ExecutionResult result)
         {
-            context.Response.ContentType = "application/json";
-            context.Response.StatusCode = result.Errors?.Any() == true ? (int)HttpStatusCode.BadRequest : (int)HttpStatusCode.OK;
+            context.Response.ContentType = "application/graphql+json";
+            context.Response.StatusCode = !result.Executed ? (int)HttpStatusCode.BadRequest : (int)HttpStatusCode.OK;
```

Comment:
I prefer to invert negate conditions.

Reply:
I have no argument; if you post a code suggestion here on GitHub, I'll accept the merge.